### ISSUE TITLE
Pull to refresh automatically release without leave it.

### ISSUE THREAD

**Issue by saurabh874** [Flipkart/recyclerlistview#625]
I am using recyclerlistview but when i perform pulltorefresh it is automatically release without leave it the drag.
it should be like  that if I hold it - it will not “slip”.
is there any way to resolve it?

<RecyclerListView
              layoutProvider={layoutProvider}
              dataProvider={dataDataProvider}
              rowRenderer={rowRenderer}
              renderAheadOffset={0}
              disableRecycling
              onScroll={(event, offsetX, offsetY) => {
                try {
                  if (offsetY >= 1 * heightRatio && !shadow) {
                    setShadow(true);
                  } else if (offsetY < 0) {
                    setShadow(false);
                  }
                } catch (e) { console.log(e); }
              }}
              scrollViewProps={{
                contentContainerStyle: { paddingBottom: (isIphoneX() ? 140 : 90) },
                stickyHeaderIndices: [1],
                refreshControl: <RefreshControl
                  onRefresh={async () => {
                    setRefreshing(true);
                    await getCampaignFilterData();
                    setRefreshing(false);
                  }}
                  refreshing={isRefreshing}
                  tintColor={theme.colors.pullToRefreshColor}
                />,
                showsVerticalScrollIndicator: false,
              }}
            />

**Comment by timurridjanovic** [comment]
Yeah, thats a pretty annoying bug.. unfortunately I feel like this repo is not being maintained anymore..

**Comment by saurabh874** [comment]
@timurridjanovic  did you use any hack to resolve this issue?

**Comment by timurridjanovic** [comment]
@saurabh874 haven't found any yet.. @naqvitalha can you point me to which file I should to look at for the pull to refresh? I'll try and submit a fix.

**Comment by timurridjanovic** [comment]
@saurabh874 found a fix. Here is my fix on another board:
https://github.com/facebook/react-native/issues/31723
